### DISCUSSION TITLE
Status of pg_tle on supabase?

### DISCUSSION THREAD

**Discussion by magick93** [supabase/supabase#21915] [Questions]
Hello

What is the status of the use of pg_tle on supabase, and database.dev?

Am I able to write, publish and use my own pg_tle extensions on supabase?

If not, what is the plans here?

**Comment by GaryAustin1** [comment]
I was able to generate couple of TLE extensions a few months ago.  I followed the instructions and got them working.
https://github.com/GaryAustin1/custom-properties/tree/main/tle-custom-properties

**Reply by magick93** [reply]
Thanks Gary

BTW - I found your [performance results](https://github.com/GaryAustin1/RLS-Performance) to be very interesting!
